src/test/euler/mocks/EulerETokenMock.sol

```diff
@@ -7,7 +7,7 @@ import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
 import {EulerMock} from "./EulerMock.sol";
 import {IEulerEToken} from "../../../euler/external/IEulerEToken.sol";
 
-contract EulerETokenMock is IEulerEToken, ERC20("EulerETokenMock", "eMOCK", 18) {
```

Comment:
Had to do this because there was a problem with `balanceOf`, which was in both `IEulerEToken` and `ERC20`, but we couldn't override any